swap_meet/vendor.py

```diff
@@ -1,2 +1,91 @@
+from .item import Item
 class Vendor:
-    pass
+    def __init__(self, inventory=[]):
+        self.inventory = inventory
```

Comment:
careful with side effects here! when you pass in a data structure as a constructor parameter, all instances will use the same data structure instead of making its own.

```suggestion
    def __init__(self, inventory=None):
        if inventory == None:
            self.inventory = []
        else:
            self.inventory = inventory
```

Reply:
updated as
`    def __init__(self, inventory=None):
        self.inventory = inventory or []`
thank you!